File: Algorithmique appliquée/chapitre3/exercice1.cpp
/* programme racine (exercice 1)
*  but : permet de saisie plusieurs valeurs et d'afficher à chaque fois la racine carrée
*  date : 3/10/2016
*/

#include <iostream>
#include <math.h>

using namespace std;

int main(int argc, char *argv[]){

    unsigned int nb;

    cout << "Entrez une valeur (0 pour finir) = ";
    cin >> nb;

    while(nb!= 0){
        cout << "racine carr\x82e = " << sqrt(nb);
        cout << endl << "Entrez une valeur =";
        cin >> nb;
    }
    return 0;
}

File: Algorithmique appliquée/chapitre3/python/exercice1.py
# -*-coding:utf-8 -*
# Programme exercice 1
# permet de saisie plusieurs valeurs et d'afficher à chaque fois la racine carrée
# date : 3/10/2016

import os
import math


# boucle sur la saisie des valeurs
nb = int(input("Entrez une valeur (0 pour finir) = "))
while nb != 0 :
    # affichage de la racine carrée
    print("racine carr\x82e = ", math.sqrt(nb))
    # saisie d'une nouvelle valeur
    nb = int(input("Entrez une valeur (0 pour finir) = "))


os.system("pause")
